agallio/ina-covid-bed
# Enhance(prov): filter by alias

# Description
Add filter by province alias

# Current Behavior
Not have been filtered by provincie alias

# New Behavior
Can use provincie alias as filter


# How to Test
- type "dki" on search -> "Jakarta" will appear
- type "ntt" on search -> "Nusa Tenggara Timur" will appear
